azure-pipelines.yml

```diff
@@ -150,6 +159,23 @@ jobs:
         # "PerTest" corresponds to individual test running time (i.e. __TestTimeout).
         timeoutPerTestInMinutes: 10
 
+# Pri1 (Pull Request, Outerloop)
+- ${{ if and(eq(variables['System.TeamProject'], 'public'), eq(variables['Build.Reason'], 'PullRequest'), eq(variables['Build.DefinitionName'], 'coreclr-outerloop')) }}:
```

Comment:
As discussed, the plan is to eventually introduce a new build definition for each group of scenarios we want to run, something like coreclr-outerloop-gcstress. That is, at least until we get parameterized pipelines. We also should stop duplicating the test job once .yml gives us better support for making the conditions more concise, maybe with the feature described in https://github.com/Microsoft/azure-pipelines-yaml/pull/46/files?short_path=d693834#diff-d6938347e5645f4cd8add8c9f6125e18.